metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -100,6 +100,19 @@ record DashboardInfo includes CustomProperties, ExternalReference {
   }
   datasetEdges: optional array[Edge]
 
+  /**
+   * Dashboards included by this dashboard.
+   * Some dashboard entities (e.g. PowerBI Apps) can contain other dashboards.
+   */
+  @Relationship = {
+    "/*": {
+      "name": "Contains",
+      "entityTypes": [ "dashboard" ],
+      "isLineage": true
+    }
+  }
+  dashboards: array[Urn] = [ ]
```

Comment:
Should this be a list of edges rather than list of urns, to support time-based lineage and attribution? Time-based seems unnecessary here but I could see value in attribution

Actually, I think both are unnecessary, but leaving this here for discussion

Reply:
will just move to the Edge type, which should give us everything we need